dateparser_cli/fasttext_manager.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
+import urllib.request
+import os
+import logging
+
+from .utils import date_parser_model_home, check_if_date_parser_model_home_exists_else_create
+from .exceptions import FastTextModelNotFoundException
+
+
+def fasttext_downloader(model=None):
+    check_if_date_parser_model_home_exists_else_create()
+
+    model_url = {
+        "small": "https://dl.fbaipublicfiles.com/fasttext/supervised-models/lid.176.ftz",
+        "large": "https://dl.fbaipublicfiles.com/fasttext/supervised-models/lid.176.bin"
+    }
+
+    if not model:
+        message = "No model name passed Supported models are: {}".join(model_url.keys())
+        raise FastTextModelNotFoundException(message)
+    elif model[0] in model_url:
```

Comment:
``model`` is a string, we shouldn't use ``model[0]`` here and below

Reply:
@avishpoddar I see. I think the simplest fix would be to change this to ``_cli_functions_map[args[0]](*args[1:])`` - note the added star. Also I'd remote the ``None`` default from the ``model`` argument, as we don't really do anything useful in this case. But if you want to improve this code further and give better error messages to the user, you can use https://docs.python.org/3/library/argparse.html or https://click.palletsprojects.com/ libraries to implement command line argument parsing.

Reply:
Thanks, now upgraded to `argparse`